src/modules/WebglDistort/Module.js

```diff
@@ -0,0 +1,230 @@
+/*
+* Resolves Fisheye Effect
+*/
+module.exports = function DoNothing(options, UI) {
+
+    var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+
+    var output;
+    var fx = require('./glfx')
```

Comment:
Why is it added to npm if it is included from the module folder?

Reply:
@tech4GT pointed out how the npm distribution wasn't working for him, so he put the script in. For now, the script is utilised but let's not remove the npm dist either

Reply:
Ok